docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -103,19 +94,25 @@ Codacy will use the updated configurations on the next analysis.
     docs/repositories-configure/codacy-configuration-file.md (list of tool short names to use on the Codacy configuration file)
 -->
 
-Codacy supports configuration files for several tools. To use a configuration file for your static analysis tool:
+!!! note
+    -   Once set, Codacy uses a configuration file even if you exclude it from Codacy analysis [using the Codacy UI](ignoring-files.md) or a [Codacy configuration file](codacy-configuration-file.md#syntax-for-ignoring-files).
+
+    -   When you update pattern settings using a configuration file, Codacy may display outdated messages for issues identified previously by those patterns.<!-- TODO Can we remove this? If not, can we clarify when this happens and if this is temporary?  -->
```

Comment:
Maybe we can move this to after the procedure. Also, I would keep the "For performance reasons".

Reply:
This sentence is still lost [by the end of the page](https://github.com/codacy/docs/pull/1933/files#diff-8b5b34d427af25055560dd40e0c14ca5d6e7e029739f9870df33f810d6435ee1R333)...  😅